spec/models/early_career_payments/eligibility_spec.rb

```diff
@@ -1289,7 +1289,7 @@
         expect(EarlyCareerPayments::Eligibility.new(eligible_itt_subject: :none_of_the_above)).to be_valid(:"eligible-itt-subject")
       end
 
-      it "is valid when the value for 'eligible_itt_subject' is one of 'chemistry, foreign languages, mathematics or physics'" do
+      it "is valid when the value for 'eligible_itt_subject' is one of 'chemistry, languages, mathematics or physics'" do
         expect(EarlyCareerPayments::Eligibility.new(eligible_itt_subject: :chemistry)).to be_valid(:"eligible-itt-subject")
         expect(EarlyCareerPayments::Eligibility.new(eligible_itt_subject: :physics)).to be_valid(:"eligible-itt-subject")
         expect(EarlyCareerPayments::Eligibility.new(eligible_itt_subject: :foreign_languages)).to be_valid(:"eligible-itt-subject")
```

Comment:
I'd probably add `expect(EarlyCareerPayments::Eligibility.new(eligible_itt_subject: :languages)).to be_invalid(:"eligible-itt-subject")` just to emphasise the fact they've diverged